hyrax.gemspec

```diff
@@ -67,7 +67,7 @@ EOF
   spec.add_dependency 'active_attr', '~> 0.9.0'
   spec.add_dependency 'redlock', '>= 0.1.2'
   spec.add_dependency 'retriable', '>= 2.9', '< 4.0'
-  spec.add_dependency 'active-fedora', '>= 11.1.3'
+  spec.add_dependency 'active-fedora', '~> 11.2'
```

Comment:
@darrenleeweber when you specify `~> 11.2` that means any `11.x` where `x >= 2`.  So this  is forcing people running `11.1.x` to update.  Is there any changes we need in hyrax that requires this update at this time?